.github/workflows/cd-executor.yml

```
@@ -1,24 +1,58 @@
 
 env:
   BALLERINA_VERSION: 2201.12.10
 
 jobs:
   release:
+    if: ${{ inputs.release_target != 'fhir-server' }}
     runs-on: ubuntu-latest
 
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)